src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -1223,6 +1231,116 @@ export default class SourceFileLinter {
 		}
 	}
 
+	#analyzeModelDataTypes(node: ts.NewExpression | ts.CallExpression) {
+		const originalFilename = this.#metadata?.xmlCompiledResource;
+		// Do not process xml-s. This case would be handled separately withing the BindingParser
+		if (originalFilename && [".view.xml", ".fragment.xml"].some((ending) => originalFilename.endsWith(ending))) {
```

Comment:
I wonder in which (future) case, we would not have an `*.xml` filename stored in the `xmlCompiledResource` metadata. Isn't a boolean sufficient here?

Reply:
But then we'll need to resolve the filename. I don't mind either, but this one seems very convenient, at least for this case